include/constants.h

```diff
@@ -11,6 +11,9 @@ namespace reinforcement_learning {  namespace name {
       const char *const  VW_CMDLINE              = "vw.commandline";
       const char *const  VW_POOL_INIT_SIZE       = "vw.pool.init.size";
       const char *const  INITIAL_EPSILON         = "initial_exploration.epsilon";
+      const char *const  LEARNING_MODE = "rank.learning.mode";
+      const char *const  LEARNING_MODE_ONLINE = "rank.learning.mode.online";
+      const char *const  LEARNING_MODE_IMITATION = "rank.learning.mode.imitation";
```

Comment:
[nit] spacing

Reply:
yes. I just fixed the constants in the tests as well. It was missed from pushing.

---
In reply to: [387881109](https://github.com/VowpalWabbit/reinforcement_learning/pull/129#discussion_r387881109) [](ancestors = 387881109)